pax-warp-core/src/test/java/org/ops4j/pax/warp/core/command/AbstractCommandRunnerTest.java

```diff
@@ -195,15 +199,20 @@ public void test06ShouldInsertData() throws  SQLException, IOException {
     public void test07ShouldDropColumnAndIndex() throws SQLException, IOException {
         runDropChangeSet();
     }
-    
+
+    @Test
+    public void test08ShouldReimportStructure() throws  SQLException, IOException {
+        dropAndCreateDatabase();
+        reimportStructure();
+    }
+
     @Test
     public void test09ShouldRenameAndDropTable() throws IOException, SQLException {
         runRenameAndDropTableChangeSet();
     }
 
     @Test
-    public void test08ShouldReimportStructure() throws  SQLException, IOException {
-        dropAndCreateDatabase();
-        reimportStructure();
+    public void test10ShouldRunSqlInSelectedDbms() throws IOException, SQLException {
+        runRunSqlChangeSet();
```

Comment:
There should be an assertion that the statement is run only for the selected databases.

This assertion would have been triggered by the incorrect comparison mentioned above.